python/pyarrow/tests/test_csv.py

```diff
@@ -326,7 +326,7 @@ def test_write_options():
     opts = cls()
 
     check_options_class(
-        cls, include_header=[True, False])
+        cls, include_header=[True, False], delimiter=[",", "\t", "|"])
```

Comment:
Can we add a case to https://github.com/apache/arrow/blob/db2401c53f63643b06cab4c683cc086b915ec5c4/python/pyarrow/tests/test_csv.py#L1875-L1880 ?

Reply:
made the change.